build/ci/phase-template.yml

```diff
@@ -33,13 +33,16 @@ phases:
           ${{ insert }}: ${{ parameters.customMatrixes }}
       ${{ insert }}: ${{ parameters.queue }}
     steps:
+    - ${{ if eq(parameters.name, 'MacOS') }}:
+      - script: brew update && brew install libomp && brew link libomp --force
+        displayName: Install build dependencies
     - script: $(_buildScript) -$(_configuration) -buildArch=$(_arch)
       displayName: Build
-    - script: $(_buildScript) -- /t:DownloadExternalTestFiles /p:IncludeBenchmarkData=$(_includeBenchmarkData)
-      displayName: Download Benchmark Data
     - ${{ if eq(parameters.name, 'MacOS') }}:
-      - script: brew update && brew install libomp mono-libgdiplus gettext && brew link gettext --force
+      - script: brew update && brew install mono-libgdiplus gettext && brew link gettext --force
```

Comment:
Why split this into 2 steps?  Why not just `brew install` all the stuff in one step?

Reply:
So I was differentiating between the build dependency and runtime dependencies. Its minor - so I will move to a single step.

---
In reply to: [265170687](https://github.com/dotnet/machinelearning/pull/2867#discussion_r265170687) [](ancestors = 265170687)